External/FEXCore/include/FEXCore/IR/IREmitter.h

```diff
@@ -40,7 +40,11 @@ friend class FEXCore::IR::PassManager;
 
   IRPair<IROp_Constant> _Constant(uint8_t Size, uint64_t Constant) {
     auto Op = AllocateOp<IROp_Constant, IROps::OP_CONSTANT>();
-    Op.first->Constant = Constant;
+    uint64_t Mask = 1ULL << Size;
+    if (Size == 64)
```

Comment:
Branchless mask calculation:

`Mask = ~0ULL >> (Size - 64);`

Works for all sizes except 0, which we don't care about here.